I hate github, I hate merge conflicts, and I hate coding (Smes edit seriously this time)

AnthonyJNgo opened the pull request:
:cl:
del: The SMES in engineering has been removed
tweak: The wiring around the main SMES has been altered slightly. Singularity and Tesla should no longer go loose if you try setting it up later in round.
/:cl:

Surprisingly this took four hours or so to get right. Nice. On the bright side I learned some new git commands.

![smesedit](https://user-images.githubusercontent.com/38970088/67452211-8131ee00-f5f0-11e9-839c-c349a50357b0.png)

Maroonenahmoron commented:
This is important, +1

AnthonyJNgo commented:
I spent four hours fixing that wire and then five hours after that doing nothing, what has happened to my life

AnthonyJNgo commented:
@steamport You’re talking about the hallway in between the engine and SMES? I didn’t touch anything there, I left it alone. 

If you’re talking about the blue circle, that’s incorrect. There are still wires underneath the SMES. The wires in the blue circle were wired the same way before, just a bit more inefficiently.

yoyobatty commented:
I don't want that SMES in port-engineering gone, that's a useful backup if the other die/explode. There's no sense in removing it as it wasn't causing issues. Also anytime stuff gets removed, it really needs to be voted on as it is a light balance change to the available power at round start.

AnthonyJNgo commented:
@yoyobatty 

Why I removed the single SMES:  #12195 

Why I wired it how I did: #11835 

Are you referring to the small hallway in between the old SMES and the 4SMES room? If that's the case, it is only "hotwiring" because I removed the singular SMES. The problem is before it would charge the singular SMES, however, the Singularity/Tesla will go loose because it is not charging the engine.

The engine **needs** to be hotwired, as soon as it starts producing power it needs to be receiving power so it doesn't go instaloose, and the SMES need to loop back around and connect to it to continue supplying it power in the event that it stops supplying power itself for whatever reason e.g. sabatoge.

yoyobatty commented:
Sounds like shit engineers and also you haven't really configured the grid properly, again reference other stations, if you want I can fix it myself, also maroons issue can easily be fixed, you don't fix a problem by removing it completely.

yoyobatty commented:
This was resolved in my PR that also fixed general cable behaviour.
